pkg/docker/docker.css

```diff
@@ -75,30 +75,32 @@
     padding-right: 8px;
 }
 
-.container-col-name {
-    width: 20%;
-}
-
-.container-col-image {
-    width: 20%;
-}
-
-.container-col-command {
-    width: 15%;
+.container-col-cpu,
+.container-col-memory-graph,
+.container-col-memory-text {
+    display: none !important;
```

Comment:
Patternfly has this selector: `table td[class*="col-"]` which accidentally matches our class names.  Grrr.